coverage.py

```diff
@@ -37,20 +37,20 @@ def to_coverage_record(self):
 
 class CoverageProvider(object):
 
-    """Run Editions from one DataSource (the input DataSource) through
-    code associated with another DataSource (the output
-    DataSource). If the code returns success, add a CoverageRecord for
-    the Edition and the output DataSource, so that the record
-    doesn't get processed next time.
+    """Run Identifiers of certain types (the input_identifier_types)
+    through code associated with a DataSource (the
+    `output_source`). If the code returns success, add a
+    CoverageRecord for the Edition and the output DataSource, so that
+    the record doesn't get processed next time.
     """
 
-    def __init__(self, service_name, input_sources, output_source,
+    def __init__(self, service_name, input_identifier_types, output_source,
```

Comment:
Consistently focus on identifiers, which is what we store in the CoverageRecord table. Instead of mapping a number of data sources to one other data source, we map a number of identifier types to a data source.